mk/vs/Packet++.vcxproj.filters

```diff
@@ -144,6 +144,9 @@
     <ClInclude Include="..\..\Packet++\header\SomeIpLayer.h">
       <Filter>Header Files</Filter>
     </ClInclude>
+    <ClInclude Include="..\..\Packet++\header\SomeIpTpLayer.h">
```

Comment:
If we move `SomeIpTpLayer` to `SomeIpLayer.h`/`SomeIpLayer.cpp`  the changes in VS files aren't necessary (both this file as well as the other VS files)